assets/scss/look-and-feel/gov-uk-frontend-toolkit-backward-compatable.scss

```diff
@@ -0,0 +1,35 @@
+/* Hide for both screenreaders and browsers */
```

Comment:
Sorry minor typo in the file name `compatible` - might have to update references in DN and RFE though